src/receive_imf.rs

```diff
@@ -429,7 +429,7 @@ async fn get_to_and_past_contact_ids(
                             context,
                             &mime_parser.recipients,
                             to_member_fingerprints,
-                            chat_id,
+                            None,
```

Comment:
Just to be more explicit, otherwise one need to scroll up to see that `chat_id == None`